modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="getting-detailed-information-about-a-target_{context}"]
+= Getting detailed information about a target
+
+The Targeting UI provides detailed information about HTTP endpoints that are targeted for metrics scraping.
+
+.Prerequisites
+
+* You have access to the cluster as an administrator for the project for which you want to view metrics targets.
+
+.Procedure
+
+*To view detailed information about a target in the Administrator perspective*:
+
+. Open the {product-title} web console and navigate to the *Observe* -> *Targets* -> *Metrics Targets* page.
+
+. Optional: Search for a target by name using the *Text* field in the search list.
+
+. Optional: Search for a target by label using the *Label* field in the search list.
+
+. Optional: Filter the targets by status and source by selecting filters in the *Filter* list.
+
+. Optional: Sort the targets by clicking one or more of the *Endpoint*, *Status*, *Namespace*, *Last Scrape*, and *Scrape Duration* column headers.
+
+. Click the URL in the *Endpoint* column for a target to navigate to its *Target Details* page. This page provides information about the target, including:
++
+--
+** The endpoint URL being scraped for metrics
+** The current status (*Up* or *Down*) of the target
+** Labels attached to the target
+** A link to the component being monitored
+** A link to the namespace
+** The most recent time that the target was scraped for metrics
+--
+
```

Comment:
change to below, follow the rule: from left to right, from top to bottom:
** The endpoint URL being scraped for metrics
** The current status (*Up* or *Down*) of the target
** A link to the namespace
** A link to the component being monitored
** Labels attached to the target
** The most recent time that the target was scraped for metrics

Reply:
fixed